spec/factories/application_forms.rb

```diff
@@ -260,6 +271,12 @@
       english_language_provider_reference { "reference" }
     end
 
+    trait :with_english_language_provider_from_seed_data do
+      english_language_proof_method { "provider" }
+      english_language_provider { providers_from_seed_data.sample }
```

Comment:
```suggestion
      english_language_provider { EnglishLanguageProvider.all.sample }
```

Your code works, but I wonder if it's enough that we can do this. It's unlikely that we'd ever have any more providers since there's no means to create them other than in the Rails console or running the seeds.